app/controllers/users_controller.rb

```diff
@@ -58,7 +58,7 @@ def unlock
     EventLog.record_event(@user, EventLog::MANUAL_ACCOUNT_UNLOCK, initiator: current_user)
     @user.unlock_access!
     flash[:notice] = "Unlocked #{@user.email}"
-    redirect_to :back
+    redirect_back(fallback_location: root_path)
```

Comment:
What is root_path for signon? Imagine this should be the GOV.UK Homepage?

Reply:
`root_path` is the signon homepage